pages/common/vue-init.md

```diff
@@ -0,0 +1,18 @@
+# vue-init
+
+> Legacy init command.
+> Require `vue-cli@2` or `@vue/cli` with `@vue/cli-init`.
+> Official templates include: webpack, webpack-simple, browserify, browserify-simple, simple.
```

Comment:
Perhaps this will read better?
```suggestion
> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
```